### DISCUSSION TITLE
Teleport trusted cluster fails after a period of time

### DISCUSSION THREAD

**Discussion by alanlubold** [gravitational/teleport#7846] [Q&A]
I had a device running Teleport as a trusted cluster, and there were a total of three nodes in the cluster, including the bastion. This all worked fine for several months, and then it stopped working. I believe I could fix this issue by deleting the /var/lib/teleport folder, but what I'd like is to understand why it failed, if that is possible. The cluster is in a remote location and it is difficult to access physically. Teleport was the only remote access to the devices. 

Other trusted clusters on the Teleport server continued to work fine during this time. 

As far as I know, there was no power loss or reboot of the device. 

Here are the config files, with some names changed for security.

[support.txt](https://github.com/gravitational/teleport/files/6956984/support.txt)

Thank you,

Alan

**Comment by webvictim** [comment]
Did you upgrade your root cluster during this time?

**Comment by alanlubold** [comment]
No.

 

Alan Lubold

 

Did you upgrade your root cluster during this time?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
